src/custom/components/NotificationBanner/index.tsx

```diff
@@ -42,12 +44,25 @@ const BannerContainer = styled.div`
 `
 export default function NotificationBanner(props: BannerProps) {
   const [isActive, setIsActive] = useState(props.isVisible)
+  const [changeOnPropStore, setChangeOnProp] = useState(props.changeOnProp)
```

Comment:
Keep the same name on property and setter

```suggestion
  const [changeOnProp, setChangeOnProp] = useState(props.changeOnProp)
```